### PR TITLE
Update various translation files

### PR DESCRIPTION
**Pull request by teolemon**

### CODE REVIEW

File: po/openbeautyfacts/nl.po [hunk 1]
```diff
@@ -56,12 +56,12 @@ msgid "OpenBeautyFacts"
...
-msgid "This product page is not complete. You can help to complete it by editing it and adding more data from the photos we have, or by taking more photos using the app for <a href=\"https://play.google.com/store/apps/details?id=org.openbeautyfacts.scanner\">Android</a> or <a href=\"https://apps.apple.com/us/app/open-beauty-facts/id1122926380?mt=8\">iPhone/iPad</a>. Thank you!"
-msgstr "Deze productpagina is niet compleet. Je kunt helpen door het product te bewerken en meer gegevens van bestaande foto's toe te voegen, of door meer foto's te maken met de app voor <a href=\"https://play.google.com/store/apps/details?id=org.openbeautyfacts.scanner&hl=nl\">Android</a> of <a href=\"https://apps.apple.com/nl/app/open-beauty-facts/id1122926380?mt=8\">iPhone/iPad</a>. Bedankt!"
+msgid "This product page is not complete. You can help to complete it by editing it and adding more data from the photos we have, or by taking more photos using the app for <a href=\"https://play.google.com/store/apps/details?id=org.openbeautyfacts.scanner&hl=en\">Android</a> or <a href=\"https://apps.apple.com/us/app/open-beauty-facts/id1122926380\">iPhone/iPad</a>. Thank you!"
+msgstr ""
```
Comment (teolemon) [inline po/openbeautyfacts/nl.po]:
```suggestion
msgstr "Deze productpagina is niet compleet. Je kunt helpen door het product te bewerken en meer gegevens van bestaande foto's toe te voegen, of door meer foto's te maken met de app voor <a href=\"https://play.google.com/store/apps/details?id=org.openbeautyfacts.scanner&hl=nl\">Android</a> of <a href=\"https://apps.apple.com/nl/app/open-beauty-facts/id1122926380\">iPhone/iPad</a>. Bedankt!"
```

File: po/openbeautyfacts/nl.po [hunk 2]
```diff
@@ -56,12 +56,12 @@ msgid "OpenBeautyFacts"
...
+msgid "<a href=\"https://world.openfoodfacts.org/donate-to-open-food-facts\"><img src=\"https://static.openfoodfacts.org/images/svg/donate-icon.svg\" alt=\"Donate to Open Beauty Facts\" /></a><p><<site_name>> is made by a non-profit association, independent from the industry. It is made for all, by all, and it is funded by all. You can support our work by <a href=\"https://world.openfoodfacts.org/donate-to-open-food-facts\">donating to Open Beauty Facts</a> and also by <a href=\"https://www.lilo.org/fr/open-food-facts/?utm_source=open-food-facts\">using the Lilo search engine</a>.<br/><b>Thank you!</b></p>"
+msgstr ""
```
Comment (teolemon) [inline po/openbeautyfacts/nl.po]:
```suggestion
msgstr "msgstr "<a href=\"https://world-nl.openfoodfacts.org/donate-to-open-food-facts\"><img src=\"https://static.openfoodfacts.org/images/svg/donate-icon.svg\" alt=\"Doneer aan Open Beauty Facts\" /></a><p><<site_name>> wordt gemaakt door een non-profit associatie, onafhankelijk van de industrie. Het is gemaakt voor iedereen, en wordt gefinancierd door iedereen. Je kan ons werk ondersteunen door <a href=\"https://world-nl.openfoodfacts.org/donate-to-open-food-facts\"> te doneren aan Open Beauty Facts</a> en ook door <a href=\"https://www.lilo.org/fr/open-food-facts/?utm_source=open-food-facts\">de Lilo zoekmachine te gebruiken</a>.<br/><b>Bedankt!</b></p>"
```

File: po/openbeautyfacts/nl.po [hunk 3]
```diff
@@ -56,12 +56,12 @@ msgid "OpenBeautyFacts"
...
+msgstr "msgstr "<a href=\"https://world-nl.openfoodfacts.org/donate-to-open-food-facts\"><img src=\"https://static.openfoodfacts.org/images/svg/donate-icon.svg\" alt=\"Doneer aan Open Beauty Facts\" /></a><p><<site_name>> wordt gemaakt door een non-profit associatie, onafhankelijk van de industrie. Het is gemaakt voor iedereen, en wordt gefinancierd door iedereen. Je kan ons werk ondersteunen door <a href=\"https://world-nl.openfoodfacts.org/donate-to-open-food-facts\"> te doneren aan Open Beauty Facts</a> en ook door <a href=\"https://www.lilo.org/fr/open-food-facts/?utm_source=open-food-facts\">de Lilo zoekmachine te gebruiken</a>.<br/><b>Bedankt!</b></p>"
```
Comment (teolemon) [inline po/openbeautyfacts/nl.po]:
```suggestion
msgstr "<a href=\"https://world-nl.openfoodfacts.org/donate-to-open-food-facts\"><img src=\"https://static.openfoodfacts.org/images/svg/donate-icon.svg\" alt=\"Doneer aan Open Beauty Facts\" /></a><p><<site_name>> wordt gemaakt door een non-profit associatie, onafhankelijk van de industrie. Het is gemaakt voor iedereen, en wordt gefinancierd door iedereen. Je kan ons werk ondersteunen door <a href=\"https://world-nl.openfoodfacts.org/donate-to-open-food-facts\"> te doneren aan Open Beauty Facts</a> en ook door <a href=\"https://www.lilo.org/fr/open-food-facts/?utm_source=open-food-facts\">de Lilo zoekmachine te gebruiken</a>.<br/><b>Bedankt!</b></p>"
```

**Comment by hangy** [comment]
Why is it possible to merge PRs with failing tests? @stephanegigandet @teolemon 😕

**Pull request merged**